include/gui/lvgl_mem.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2018 Jan Van Winkel <jan.van_winkel@dxplore.eu>
```

Comment:
is this a public API of Zephyr? We should not have things under include/ that are not public APIs. We are in the process of cleaning up and we should not add new non-public headers in here.

Reply:
No it is not a public API, moved header file to lib/gui/lvgl